src/commands/difutil.rs

```diff
@@ -8,27 +8,27 @@ macro_rules! each_subcommand {
         $mac!(difutil_bundle_sources);
         $mac!(difutil_find);
         $mac!(difutil_check);
-        $mac!(difutil_id);
     };
 }
 
-pub fn make_app<'a, 'b: 'a>(mut app: App<'a, 'b>) -> App<'a, 'b> {
+pub fn make_app(mut app: Command) -> Command {
```

Comment:
should we rename all references from `app` -> `command` to match the new clap API? We can do that in a follow up PR as well. So `make_app` -> `make_command`